### PR TITLE
Flake pod sandbox errors pinging container registry on azure.

### PR DESCRIPTION
**Pull request by dgoodwin**
Happening often enough that we need to ignore on Azure. Likely similar
to the ErrImagePull problems we also handle here.

### CODE REVIEW

**Comment by stbenjam** [comment]
/lgtm
/approve
/hold

Let's test azure, but the change looks fine to me.  Feel free to remove the hold once azure comes back

File: pkg/synthetictests/networking.go [hunk 1]
```diff
@@ -70,6 +86,12 @@ func testPodSandboxCreation(events monitorapi.Intervals) []*junitapi.JUnitTestCa
 			// See https://github.com/openshift/origin/blob/93eb467cc8d293ba977549b05ae2e4b818c64327/test/extended/networking/whereabouts.go#L52
 			continue
 		}
+		if strings.Contains(event.Message, "pinging container registry") && strings.Contains(event.Message, "i/o timeout") {
+			if infra != nil && infra.Spec.PlatformSpec.Azure != nil {
```
Comment (deads2k) [inline pkg/synthetictests/networking.go:90]: Status holds the authoritative: "this is the state of the cluster" using the discriminated union type: https://github.com/openshift/api/blob/c689b3938fab2ff48f7735ac410e7565d6e53426/config/v1/types_infrastructure.go#L276

**Pull request merged**